ci/docker/install/ubuntu_onnx.sh

```diff
@@ -27,8 +27,17 @@ set -e
 set -x
 
 echo "Installing libprotobuf-dev and protobuf-compiler ..."
-apt-get install -y libprotobuf-dev protobuf-compiler
+apt-get install -y autoconf automake libtool curl make g++ unzip
```

Comment:
The CI failed at 'Sanity Check' for this commit. When the CI tries to execute this apt-get, it is returning with a HTTP 404. But I have tried this script with an EC2 instance and there it installs "curl" and "libtool" fine.
@marcoabreu Could you please point out if I am missing something here?
```bash
The following NEW packages will be installed:
  curl libtool
0 upgraded, 2 newly installed, 0 to remove and 22 not upgraded.
Need to get 332 kB of archives.
After this operation, 1254 kB of additional disk space will be used.
Err:1 http://security.ubuntu.com/ubuntu xenial-security/main amd64 curl amd64 7.47.0-1ubuntu2.6
  404  Not Found [IP: 91.189.88.152 80]
Get:2 http://archive.ubuntu.com/ubuntu xenial/main amd64 libtool all 2.4.6-0.1 [193 kB]
Err:1 http://security.ubuntu.com/ubuntu xenial-security/main amd64 curl amd64 7.47.0-1ubuntu2.6
  404  Not Found [IP: 91.189.88.152 80]
```

Reply:
Could be possible that the apt server had some problems. I've retriggered it.